Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG GOARCH="amd64"
+
+FROM golang:1.24 AS builder
```

Comment:
be aware of kubernetes and golang versions relation, you can find the relation in this filel https://github.com/kubernetes/kubernetes/blob/master/.go-version in each branch https://github.com/kubernetes/kubernetes/blob/master/.go-version, it is 1.23.6 in 1.32 ... it is ok but sporadically there are changes in golang that can affect